CONTRIBUTING.md

```diff
@@ -89,4 +67,85 @@ https://xxxxx.biterg.io/app/kibana#/dashboard/2e968fe0-b1bb-11e8-8aac-ef7fd4d8cb
 identifier ---> 2e968fe0-b1bb-11e8-8aac-ef7fd4d8cbad
 ```
 
-After the execution of kidash you should get two files, one with the index pattern and one with the panel itself. The second one is the one ready to be shared with the community!
+After the execution of kidash you should get at least two files, one per each index pattern used in the panel and one with the panel itself. You should put them under `<collection_name>/` directory.
+
+### Adding or updating documentation
+
+If you are adding a new panel or updating an existing one, you will be required to document it. Bug fixes usually don't need documentation updates.
```

Comment:
You should say the documentation is generated with Jekyll and add a link to the format Jekyll expects (in case someone doesn't know about Jekyll).